runtime/zephir.c

```diff
@@ -176,9 +175,9 @@ static void zephir_parse_file(const char *file_name TSRMLS_DC)
 		return;
 	}
 
-	if ((len = php_stream_copy_to_mem(stream, &contents, maxlen, 0)) > 0) {
+	if (contents = php_stream_copy_to_mem(stream, PHP_STREAM_COPY_ALL, 0)) {
 
-		zephir_parse_program(&return_value, contents, len, file_name, NULL TSRMLS_CC);
+		zephir_parse_program(&return_value, contents, ZSTR_LEN(contents), file_name, NULL TSRMLS_CC);
```

Comment:
[Why I change]

`ZSTR_LEN` is similar to `contents->len`, (len is a field of `zend_string` structure)